dashboard/src/actions/operators.test.tsx

```diff
@@ -44,6 +44,26 @@ describe("checkOLMInstalled", () => {
     await store.dispatch(operatorActions.checkOLMInstalled());
     expect(store.getActions()).toEqual(expectedActions);
   });
+
+  it("dispatches error", async () => {
+    Operators.isOLMInstalled = jest.fn(() => {
+      throw new Error("nope");
+    });
+    const expectedActions = [
+      {
+        type: getType(operatorActions.checkingOLM),
+      },
+      {
+        type: getType(operatorActions.errorOperators),
+        payload: new Error("nope"),
+      },
+      {
+        type: getType(operatorActions.OLMNotInstalled),
+      },
```

Comment:
It's not obvious to me why we'd still dispatch an OLMNotInstalled after dispatching an error: we know there was an error, we don't know that the OLM is not installed?

Reply:
Well, yes, at the beginning this was me trying to be extra-verbose. `OLMNotInstalled` action is indeed not needed (even less now that this may be triggered if we are not able to verify if it's installed).